src/framework/body/BodyPanel.jsx

```diff
@@ -1,18 +1,20 @@
 
+import classNames from 'classnames';
 import React, {
   PropTypes,
 } from 'react';
-import classNames from 'classnames';
+
+import BaseBox from '../base/box/BaseBox.jsx';
 
 const BodyPanel = props => {
   const classes = classNames('bodyPanel', {
     'bodyPanel--topFlush': props.isTopFlush,
   });
 
   return (
-    <div className={classes}>
+    <BaseBox classes={classes}>
       {props.children}
-    </div>
+    </BaseBox>
   );
 };
 
```

Comment:
With this implementation it won't be possible to have a BodyPanel with rounded corners as we intended. BodyPanel needs to accept a prop `roundedCorners` and pass it to BasBox.